### ISSUE TITLE
Image to Manifest link?

### ISSUE THREAD

**Issue by azaroth42** [IIIF/iiif.io#600]
@aisaac asks in #558:
> [Mapping to EDM/DPLA is] made more difficult by the fact there is no link between an Image URI and a (canonical) manifest URL

To expand:  In info.json, there isn't a way to find which manifests the image is used in, even if the image and manifest are published by the same content provider. If there was a link there, and the image was discovered, it could be followed to find the manifest with all of the other metadata and structural information, thus allowing a viewer to provide a better UX.

This assumes the following situation:
 * There is a link to the image from some source other than the manifest
 * ... including a second or subsequent manifest ...
 * ... that is known about by the Image provider ...
 * ... that the consumer of the Image would be able to follow and render 

I think this would be good to discuss, possibly in Ghent if not before, under the discovery rubric.

**Comment by azaroth42** [comment]
Discussed in Ghent, and I think we can close with nothing to do from the IIIF end?
Yes @kestlund, @anarchivist, @aisaac?

**Comment by aisaac** [comment]
I think indeed we concluded that from the perspective of solving #558 we agreed a link to the manifest was not strictly needed in the info.json.
However I can't tell for the perspective of other discussions. I might have overheard people interested in pointing to manifest as 'prefered' ones for an image. But I've been suggesting this in advance of the meeting so might have dreamed it altogether ;-) If you don't have any record of this then let's consider the matter solved.

**Comment by anarchivist** [comment]
I seem to remember a minor discussion about this in Ghent, following @aisaac's understanding. I, for one, am interested if there's a preferred manifest (e.g. at least to signal that an image API adopter has now also started publishing manifests)...

**Comment by jpstroop** [comment]
I wasn't there for the discussion, but it occurs to me that the mechanism for exposing this info would be a [service](http://iiif.io/api/annex/services/#services). We could never force clients to do this, but we could at least add an example in the services note ([as we've done with image information](http://iiif.io/api/image/2.0/#image-information)) so that there is a recommended pattern to follow.

**Comment by anarchivist** [comment]
Yep - that's what I seem to remember as an informal suggestion (I think made by @zimeon, but I might be mistaken).

**Comment by azaroth42** [comment]
Proposal accepted to use dct:isReferencedBy for link between image and manifest, and to save isDescribedBy in case of linking from image to info.json.

**Comment by azaroth42** [comment]
Where does this get documented? Is it Image API, or just a note?  I think I prefer a note.

**Comment by anarchivist** [comment]
:+1: to a note.

**Comment by aisaac** [comment]
+1 for a note.

**Comment by zimeon** [comment]
:+1: for note too

**Comment by azaroth42** [comment]
Who wants to write up this note?

**Comment by tomcrane** [comment]
Is this a cookbook entry? Does the cookbook have a set of recipes for:

Linking from Image API to Presentation API
Linking from Image API to external metadata
Linking from external metadata to Image API
Linking from external metadata to Presentation API
Linking between Presentation API representations

**Comment by azaroth42** [comment]
:+1: to putting these sorts of things into cookbook recipes.

**Comment by azaroth42** [comment]
See #1507.
